SimplenoteUITests/EmailLogin.swift

```diff
@@ -14,7 +14,18 @@ class EmailLogin {
         backButton.tap()
     }
 
-    class func logIn(email: String, password: String) {
+    class func logIn() {
+        var testAccount = ProcessInfo.processInfo.environment["UI_TEST_ACCOUNT"]!
+
+        // Use 'default' account if test account was not passed via environment variable
+        if testAccount.isEmpty {
+            testAccount = testDataEmail
+        }
+
+        EmailLogin.logIn(testAccount, testDataPassword)
+    }
+
+    class func logIn(_ email: String, _ password: String) {
```

Comment:
This is nice to know, thanks! I previously thought that using nameless parameters is something dictated by gut feeling rather than having a more well-define rule, like not using it for weakly typed parameters.